mxcube3/routes/login.py

```diff
@@ -58,6 +60,18 @@ def signout():
 
         return redirect("/login", code=302)
 
+    @bp.route("/signout_user", methods=["POST"])
+    # @server.restrict
+    def signout_user():
```

Comment:
This is already implemented [here](https://github.com/mxcube/mxcubeweb/blob/81381c34c11f802d129e993c094e3bd267d9b74f/mxcube3/routes/ra.py#L95)

I think its about the same ?

Reply:
As I said we are going to force to sign out a user (myself) while we are not yet logged in (but authenticated). Then we have to define a new method which is not @server.restrict (?)

Reply:
Hm, I guess we could lighten up the restriction. So far we have not needed to distinguish between logged in and authenticated, but there is perhaps a need. In that case `@server.restrict` could be made to simply check authenticated or as you mention we create a something like  `@server.restrict_authenticated`. I have to admit the the later sounds a bit odd to me. From the "restrict" point of view you are either "allowed/authenticated" or not. We could perhaps pass a parameter to `@server.restrict` like `@server.restrict(allow_authenticated = True )`. However not having any restrict could allow anyone to logout users.

Does that make sense ?